### DISCUSSION TITLE
EasyLink black screen

### DISCUSSION THREAD

**Discussion by DaniilTrufanov** [cedricp/ddt4all#1004] [Q&A]
I have a 2021 Renault Duster 2
After trying to activate wireless applecar play (under [Navigation] A-IVI v1.12/ System Information $DAB1), the easylink multimedia system screen does not switch on. After setting the required configuration and pressing the key off/on reset the system rebooted, turned on (music playing, multimedia responding to the controller) but the screen remained off. I reset the system back to the original settings, but it didn't help. Could you please suggest solutions to the problem? Maybe I accidentally changed some other settings? I am attaching screenshots of System Information $DAB1 configuration.
![1](https://github.com/cedricp/ddt4all/assets/156349761/f90539c2-4b3f-44b1-b686-a99bed64603d)
![2](https://github.com/cedricp/ddt4all/assets/156349761/f070593f-ecf8-4436-9a88-69b1b3e593f3)
![3](https://github.com/cedricp/ddt4all/assets/156349761/525b15b0-bd5d-4b88-a21b-6055754861f1)
![4](https://github.com/cedricp/ddt4all/assets/156349761/aed4c449-3872-4e21-8542-11b20472e453)
![5](https://github.com/cedricp/ddt4all/assets/156349761/dfe46b0b-5ee4-49ef-a7ad-015ba8b4ccd6)

**Comment by KarelSvo** [comment]
The log files are missing.

**Reply by DaniilTrufanov** [reply]
How can I fix this? Can you help me?

**Comment by DaniilTrufanov** [comment]
The problem was solved by selecting "Attached fascia (A-IVI)" in the "Touch_screen_configuration" option.
